autonomic-administration-plugin/src/main/java/br/com/autonomiccs/autonomic/administration/plugin/AdministrationAgent.java

```diff
@@ -93,11 +94,18 @@
     @Autowired
     private VirtualMachineService virtualMachineService;
 
+    /**
+     * Time in seconds to wait for migrations. As the agent will not be doing nothing in this
+     * cluster for awhile, it sleeps {@value} seconds.
```

Comment:
The `@value` notation allows to print the value stored at the documented variable  when checking for the documentation, in this case 10.